test/helpers/mock.ts

```diff
@@ -90,14 +90,34 @@ export function withExternalServicesMock() {
 
     mockCarbon
       .intercept({
-        path: '/intensity/2023-12-01T00:00:00.000Z/2023-12-02T00:00:00.000Z',
+        path: '/intensity/2023-12-01T09:00:00.000Z/2023-12-02T10:00:00.000Z',
```

Comment:
why have we shifted by 9 hours and 10 hours respectively here? Surely the change in src/lib/services/emissionsCalculator.ts only shifts the `startTime` and only by 1 hour

Reply:
This should be reverted back, sorry this was unintentional change.